src/gpu/intel/gemm/jit/gen_kernel.cpp

```diff
@@ -478,8 +478,7 @@ status_t gen_nocopy_desc_t::select_kernel(compute::gpu_arch_t arch,
     if (b_quant.zp_ndims >= 0) problem_.bOffset = ABOffset::Calc;
     problem_.aoPtrDims = a_quant.zp_ndims;
     problem_.boPtrDims = b_quant.zp_ndims;
-    problem_.AO.layout = MatrixLayout::N;
-    problem_.BO.layout = MatrixLayout::T;
+    problem_.AO.layout = problem_.BO.layout = MatrixLayout::N;
```

Comment:
We should keep `BO.layout = MatrixLayout::T` for 1D zp, to allow for block loads.

Reply:
Done.